### ISSUE TITLE
test_boto3 and test_botocore fails due to package update (bundled six)

### ISSUE THREAD

**Issue by matysek** [pyinstaller/pyinstaller#1844]
Both Appveyor and Travis are showing failures in test_boto3 and test_botocore due to an update to both packages.
- The exe complains about missing module `html.parser`:

```
  File "/lib/python3.4/site-packages/botocore/vendored/six.py", line 79, in _import_module
    __import__(name)
ImportError: No module named 'html.parser'
LOADER: Back to parent (RC: 255)
LOADER: Doing cleanup
LOADER: Freeing archive status for /test_botocore_onedir_0/dist/test_botocore/test_botocore
```
- module `html.parser` is wrapped in `six` module.
- I think this is somehow connected to the bundled version of `six` module as `botocore.vendored.six`.
- I guess PyInstaller is not able to execute the `six` pre_safe_import hook for module `botocore.vendored.six`. Probly some doing that could fix those tests.

**Comment by matysek** [comment]
One think I do not understand. Locally these tests fail for me with Python 3.4. However, in travis they fail only with Python 3.3.

**Comment by matysek** [comment]
> One think I do not understand. Locally these tests fail for me with Python 3.4. However, in travis they fail only with Python 3.3.

Removing Travis cache did the trick to get gain stable travis build.

**Comment by nchammas** [comment]
I'm seeing this issue as well. I'm running Python 3.5.1, boto3 1.2.6, and botocore 1.4.1.

**Comment by wayne-luminal** [comment]
Same here with 

python 3.4.2/3.5.1
boto3==1.3.1
botocore==1.4.29
pyinstaller==3.2

**Comment by msirull** [comment]
I'm having the same issue. Does anyone know of any workarounds?

**Comment by wayne-luminal** [comment]
@msirull I made an unused import `import html.parser` in one of my modules and that allowed me to build successfully. Since the module deep inside the bundled `six` lib is using this to build docs, I consider it a reasonable workaround so I can keep working.

**Comment by wayne-luminal** [comment]
I also make a `hook-botocore.py` with the contents
```python
hiddenimports = [
    "html.parser"
]
```
Which worked and is a better solution than dirtying up the code with unused imports.

**Comment by thierryturpin** [comment]
or in python 2.7: 
from HTMLParser import HTMLParser

**Comment by bwoodsend** [comment]
Fixed in 007462f.
